apps/src/ImageSegmentationMain.cpp

```diff
@@ -0,0 +1,29 @@
+#include "rt/DisegniSegmenter.hpp"
+
+using namespace cv;
```

Comment:
We only allow `using namespace` when inside a class's cpp and when the namespace belongs to the project:
`using namespace rt;`

Otherwise, only namespace aliases are to be used to make code more readable:
```c++
namespace fs = boost::filesystem;
```